django-backend/fecfiler/scha_transactions/serializers.py

```diff
@@ -10,11 +11,19 @@ class SchATransactionSerializer(
     CommitteeOwnedSerializer, serializers.FecSchemaValidatorSerializerMixin
 ):
     schema_name = "SchA"
+    parent_transaction = PrimaryKeyRelatedField(
+        many=False,
+        required=False,
+        queryset=SchATransaction.objects.all(),
+    )
 
     class Meta:
         model = SchATransaction
         fields = [
-            f.name for f in SchATransaction._meta.get_fields() if f.name != "deleted"
+            f.name for f in SchATransaction._meta.get_fields() if f.name not in [
```

Comment:
I think you'll need to add
```
        foreign_key_fields = {"parent_transaction": "parent_transaction"}
```
to the meta so the correct json gets sent to the validator

Reply:
It does not seem to be necessary as `parent_transaction` is a foreign key field in the model where `report_code` was a char field that needed to be treated as a foreign key.